icons/disc-album-2.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="20" height="20" x="2" y="2" rx="2" />
```

Comment:
Maybe just a bit too big.

Reply:
> Maybe just a bit too big.

@jguddas No other way of accommodating this `disc` variant… It’s the same height as `disc-album-remove`.